Backend/src/main/java/lt/vu/ads/controllers/OrderController.java

```diff
@@ -97,6 +101,15 @@ public Order createOrder(@RequestBody Order order)
             addressController.createAddress(order.getSourceAddress());
         }
 
+        Date date = new Date();
+        order.setDate(date);
+
+        order.setEstimatedArrivalTime(orderService.calculateArrivalTime(order.getIsExpress()));
+        if (order.getSize() == null){
+            throw new CustomException("Box size is null");
```

Comment:
While getting orders from database, there shouldn't be any null sizes. So is it important to check here?

Reply:
If you talk about order.getIsExpress value, so by default it is false. So, this value will never be null.

Reply:
To check if the value is null is worth for security reasons. For example, if the UI does not have default value.